widget/group.go

```diff
@@ -131,7 +131,7 @@ func (g *groupRenderer) Objects() []fyne.CanvasObject {
 }
 
 func (g *groupRenderer) Refresh() {
-	g.label.Text = g.group.Text
```

Comment:
Changing from a Field set to a method call means that the group will be refreshed twice... That is one of the benefits for setting fields then the refresh being called after all changes are set.

Reply:
Thanks. Can we set it directly here to avoid the double refresh?

Reply:
same as above... :(  added back in 9d50247